src/ProjectSchema/SchemaProperties.tsx

```diff
@@ -0,0 +1,68 @@
+import { BsFillPencilFill, BsFillTrashFill } from "react-icons/bs";
+
+import { TypeIcon } from "../@shared";
+import * as css from "./SchemaProperties.css";
+
+import type { Schema } from "../@types/schema";
+
+type Options = {
+  schema: Schema;
+  editHandler: ({ propertyName }: { propertyName: string }) => void;
+  deleteHandler: ({ propertyName }: { propertyName: string }) => void;
+};
+
+export function SchemaProperties({
+  schema,
+  editHandler,
+  deleteHandler,
+}: Options) {
+  if (!Object.keys(schema.properties).length) {
+    return null;
+  }
+
+  // TODO any type
+  return (
+    <>
+      {Object.entries(schema.properties).map(([propertyName, data]: any) => (
+        <div className={css.field}>
+          <div className={css.fieldTypeIcon}>
+            <TypeIcon type={data.type === "string" ? "text" : data.type} />
+          </div>
+          <div className={css.fieldInfoWrapper}>
+            <div>{propertyName}</div>
+            <div className={css.fieldInfo}>
+              <div>{data.type}</div>
+              {schema.required && schema.required.includes(propertyName) && (
+                <div>required</div>
+              )}
+              {Object.entries(data).map(([key]) => (
+                <>
+                  {key === "min" && <div>min</div>}
```

Comment:
```suggestion
                  <div>{key}</div>
```

이렇게 하면은 동작이 안되나용....?

Reply:
그렇게 하면 key가 type인 경우도 함께 나타나서 저런식으로 했었어요. 근데 생각해보니
`<>{key !== "type" && <div>{key}</div>}</>`
이런식으로 key가 type인 경우만 빼주면 되겠네요~~

Reply:
오 그게 좋겠네용 😊